content/en/examples/policy/restricted-psp.yaml

```diff
@@ -22,8 +22,9 @@ spec:
     - 'projected'
```

Comment:
this policy is mutating because it sets `seccomp.security.alpha.kubernetes.io/defaultProfileName` and `apparmor.security.beta.kubernetes.io/defaultProfileName`

Reply:
Right. That's the only way to allow the default for AppArmor, but it makes sense to avoid that for seccomp so folks don't rely on it and then switch to PodSecurity.

Reply:
> That's the only way to allow the default for AppArmor

really? you can't do this:

```
apparmor.security.beta.kubernetes.io/allowedProfileNames: ',runtime/default'
```

Reply:
looks like not:

```
	profile := apparmor.GetProfileNameFromPodAnnotations(pod.Annotations, container.Name)
	if profile == "" {
		if len(s.allowedProfiles) > 0 {
			allErrs = append(allErrs, field.Forbidden(fieldPath, "AppArmor profile must be set"))
			return allErrs
		}
		return nil
	}
```

that's pretty crummy...

Reply:
I removed the seccomp default, so that users aren't relying on that mutation when they switch to PodSecurity. Unfortunately there's no way to allow the unset value with AppArmor, so the only way to allow the default is to set it explicitly. This shouldn't affect the migration, so I'd like to keep it as is. WDYT?